recipes/arize-phoenix-evals/meta.yaml

```
@@ -0,0 +1,40 @@
+{% set name = "arize-phoenix-evals" %}
+{% set version = "0.0.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/arize_phoenix_evals-{{ version }}.tar.gz
+  sha256: baa6956b6904d7f8f1d0df13f6541bd070366df554a519c41eb0a0bf49b3a61a
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8,<3.12
+    - hatchling
+    - pip
+  run:
+    - python >=3.8,<3.12
+    - pandas
+    - tqdm
+    - typing-extensions >=4.5,<5
+
+test:
```

Please check requirements using pip check.
```suggestion
test:
  requires:
    - pip
  commands:
    - pip check
```